### PR TITLE
Improve prices in dex.trades

### PR DESCRIPTION
**Pull request by k06a**
Let me know WDYT?

I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

**Comment by milkyklim** [comment]
Is possible to replace duplicate code with hop-view? Something that maps one token into another: 
```
from | to 
-----------
bETH | ETH
USDC | 1
... 
```

**Comment by mewwts** [comment]
Hey, if we want this functionality it probable makes sense to define it in a cte:
```sql
WITH prices AS (
....
)
```

File: schema/dex/trades.sql [hunk 1]
```diff
@@ -712,14 +712,54 @@ WITH rows AS (
...
+        select 'BETH', 18, '\xc0829421c1d260bd3cb3e0f06cfe2d52db2ce315', price from prices.layer1_usd WHERE symbol = "ETH" AND pa.minute = date_trunc('minute', dexs.block_time) UNION ALL
+        
+        select 'renBTC', 8, '\xeb4c2781e4eba804ce9a9803c67d0893436bb27d', price from prices.layer1_usd WHERE symbol = "BTC" AND pa.minute = date_trunc('minute', dexs.block_time) UNION ALL
+        select 'WBTC', 8, '\x2260fac5e5542a773aa44fbcfedf7c193bc2c599', price from prices.layer1_usd WHERE symbol = "BTC" AND pa.minute = date_trunc('minute', dexs.block_time) UNION ALL
+        select 'sBTC', 18, '\xfe18be6b3bd88a2d2a7f928d00292e7a9963cfc6', price from prices.layer1_usd WHERE symbol = "BTC" AND pa.minute = date_trunc('minute', dexs.block_time) UNION ALL
+
+        select 'USDT', 6, '\xfe18be6b3bd88a2d2a7f928d00292e7a9963cfc6', 1 UNION ALL
+        select 'USDC', 6, '\xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', 1 UNION ALL
+
+        select 'SAI', 18, '\x89d24a6b4ccb1b6faa2625fe562bdd9a23260359', 1 UNION ALL
```
[the remark this answers is not in the document]
Comment (k06a) [reply schema/dex/trades.sql:755]: At least let's do this for ETH to have right prices in `dex.trades`.

**Comment by k06a** [comment]
Wdyt?

**Review by Redwarrio — APPROVED** [review]

**Comment by masquot** [comment]
Thank you @k06a. Will close this one now.

Did quick research and Mooniswap and DODO have trades involving `ETH` without `usd_amount`. Created #556 to address this.

**Pull request closed without merging**